src/components/layout/ScrollToTopOnMount.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import { Component } from 'react';
+
+// https://reacttraining.com/react-router/web/guides/scroll-restoration/scroll-to-top
+class ScrollToTopOnMount extends Component<*> {
+  componentDidMount() {
+    window.scrollTo(0, 0);
+  }
+
+  render() {
+    return null;
+  }
+}
+
+export default ScrollToTopOnMount;
```

Comment:
This is a great point—the reason I decided to put this functionality in a separate component and import it to all the various pages though is because almost all of those pages are stateless functional components, and I didn't want to make them classes with state/lifecycle methods just for the purpose of scrolling to the top on mount. I stole the idea from @solomontang.